java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -734,19 +734,24 @@ private static boolean isNonNullMethod(Symbol symbol) {
     return !symbol.isUnknown() && symbol.metadata().isAnnotatedWith("javax.annotation.Nonnull");
   }
 
+  /**
+   * We do nothing for local variables without initializers, for other variables we use default value or initializer if there is one
```

Comment:
I would use passive form here. "Nothing is done..." as you and me are not really doing the computation :wink: